x/evm/keeper/dynamic_precompiles.go

```diff
@@ -46,8 +46,24 @@ func (k Keeper) GetDynamicPrecompilesInstances(
 	return addresses, activePrecompileMap
 }
 
+// GetDynamicPrecompileInstance returns the dynamic precompile instance for the given address.
+func (k Keeper) GetDynamicPrecompileInstance(
+	ctx sdk.Context,
+	params *types.Params,
+	address common.Address,
+) (vm.PrecompiledContract, bool) {
```

Comment:
Should we return here an error instead of `bool`?
Cause otherwise we don't bubble up the error on the `InstantiateERC20Precompile`